.github/workflows/master.yml

```diff
@@ -23,3 +23,31 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.FLYTE_BOT_PAT }}
           WITH_V: true
           DEFAULT_BUMP: patch
+  goreleaser:
+    name: Goreleaser
+    runs-on: ubuntu-latest
+    needs: [bump-version]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+      - name: Generate Serialize proto
```

Comment:
This is not tested. I will test it in CI.

Reply:
It's working fine and I am considering that after this workflow, The user will not use serialize command. They will use the `flytectl register` directly. I made changes in `cookbook/common/Makefile`. 

But getting error while building all in one https://github.com/evalsocket/flytesnacks/runs/2466265443?check_suite_focus=true#step:4:38041

Reply:
@EngHabu Tested ✅ https://github.com/evalsocket/flytesnacks/actions/runs/796387320

```
flytectl register files https://github.com/evalsocket/flytesnacks/releases/download/v10.7/flytesnacks-Core.tgz -d development -p flytesnacks --config config.yaml --archive
```